src/plugins/console/public/application/models/sense_editor/sense_editor.ts

```diff
@@ -466,8 +466,19 @@ export class SenseEditor {
         // no request block
         return req;
       }
-
-      const esPath = req.url;
+      const reg = /[?=&,/]/g;
+      //  we must encode url special characters without ?=&,/ symbols
+      //  but when "/" is before "d" we must encode it also
+      const esPath = req.url
```

Comment:
@hro-maker I did a bit of testing and I think we should consider the case of encoded URLs as well. I suggest something similar to [this](https://github.com/elastic/kibana/pull/128727/files#diff-149ea26afd84194c94ae0c3c9ef21bb9784ca0385ae27f61415af5a8c0919ae1R43). Also, encoding the path with `encodeURIComponent` should be sufficient.

Reply:
thank you @mibragimov,i will make this changes